src/domain/entities/Event.ts

```diff
@@ -0,0 +1,18 @@
+import { Id } from "./Base";
+import { DataValue } from "./DataValue";
+
+export type EventMetadata = {
+    event: Id;
+    orgUnit: Id;
+    program: Id;
+    programStage: Id;
+    status: "ACTIVE" | "COMPLETED" | "VISITED" | "SCHEDULED" | "OVERDUE" | "SKIPPED";
+    eventDate: string;
+    dataValues: Pick<DataValue, "dataElement" | "value">[];
```

Comment:
This `DataValue` refers to data set data values, which are not identical to program events. You can define the type directly here.